src/screens/LoginScreen.tsx

```diff
@@ -24,23 +28,29 @@ export const LoginScreen: ModuleScreen = () => {
   const buttonDisabled = stage == Stages.Login && value == ''
 
   // Increment through each stage
-  const nextStage = () => {
-    let nextStage = stage + 1
+  const setScrollStage = (nextStage: Stages) => {
+    // Updare internal state
     setStage(nextStage)
+
     // Update scorller
     loginScroll.current?.scrollTo({
       x: nextStage * dimensions.width,
       y: 0,
       animated: true,
     })
 
-    // If at loading stage then show experiment
-    // TODO: Remove when implmenting login code...
+    // If at loading stage then attempt login
     if (nextStage == Stages.Loading) {
-      const exampleExperiment = new ExperimentViewController(
-        exampleExperimentData,
-      )
-      exampleExperiment.present()
+      // Check if user is using demo login
+      if (value == 'local.demo') {
```

Comment:
That looks like something that it would be useful to document somewhere? Is the project going to have documentation in the repository or elsewhere?

Reply:
Added to ReadME - This hasn't been decided yet but we do have a tech spec.